src/visitors/rules/custom/parameterAllOf.js

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable class-methods-use-this */
+
+class parameterAllOf {
+  constructor(config) {
+    this.config = { ...config };
+    switch (this.config.level) {
```

Comment:
Do we need every custom rule to add this code? Maybe we can provide some base class with this behaviour handled. Also, where is this level used later? Is it propagated down the visitor invocations?

Also, I thing we should export constants for those 1, 2, 3, 4 numbers :)

Reply:
I wanted to make custom rules completely 0-dependency. We can just inherit from AbstractVisitor  class and don't do this manually

Reply:
Also, numbers are in constants elsewhere

Reply:
Hmm. I think 0-dep makes sense. Maybe, instead we can just keep string values and adjust other related code to accept them instead of/ additionally to number values?

Reply:
Hmmm, yes, that might work fine